cset-workflow/app/metplus_grid_stat/file/GridStat.conf

```diff
@@ -0,0 +1,88 @@
+[dir]
+FCST_BASE = {ENV[METPLUS_FCST_DIR]}
+ANA_BASE = {ENV[METPLUS_ANA_DIR]}
+OUTPUT_BASE = {ENV[CYLC_TASK_WORK_DIR]}
+MET_INSTALL_DIR = {ENV[MET_INSTALL_DIR]}
+TIME_START = {ENV[TIME_START]}
+
+[config]
+
+# Documentation for this use case can be found at
+# https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/ASCII2NC/ASCII2NC_python_embedding.html
+
+# For additional information, please see the METplus Users Guide.
+# https://metplus.readthedocs.io/en/latest/Users_Guide
+
+###
+# Processes to run
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#process-list
+###
+
+SCRUB_STAGING_DIR = False
+PROCESS_LIST = GridStat
+LOG_GRID_STAT_VERBOSITY = 4
+
+###
+# Time Info
+# LOOP_BY options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+# LEAD_SEQ is the list of forecast leads to process
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#timing-control
+###
+
+LOOP_BY = INIT
+INIT_TIME_FMT = %Y%m%dT%H
+INIT_BEG = {TIME_START}
+INIT_END = {TIME_START}
+INIT_INCREMENT = 1H
+
+#LEAD_SEQ = 0
```

Comment:
```suggestion
```

Reply:
Note that `GridStat.conf` is not intended to be used, as the site-specific cycling and science configuration files will be used instead.